src/js/percentage.js

```diff
@@ -0,0 +1,54 @@
+/*!
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.chartAuthoring.percentage");
+
+    gpii.chartAuthoring.percentage.calculate = function (value, total) {
+        value = parseFloat(value, 10);
```

Comment:
This facility needs to go into the rendering method and not the calculator or parser. In answer to cindy's question, I don't think that any real user of this system will be interested in receiving all digits of a percentage to double floating point precision. However, you can follow the behaviour advertised by https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed